.github/workflows/ci.yml

```diff
@@ -96,6 +96,8 @@ jobs:
     with:
       uploadImageAsTarball: "restate.tar"
       platforms: linux/amd64
+      # additional features added for CI validation builds only
+      features: metadata-api
```

Comment:
This is only enabled on the internal docker artifact we attach to the workflow run - same one that gets used by the SDK tests.